src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2509,6 +2509,11 @@ namespace winrt::TerminalApp::implementation
         CATCH_LOG();
     }
 
+// The current UCRT has a neat AVX2 implementation for all of the various strlen() functions.
+// This greatly benefits _PasteFromClipboardHandler, where we're calling wcslen while holding a global
+// lock onto the clipboard. MSVC treats wcslen as a builtin and so we need to disable it explicitly.
```

Comment:
This is fine as is, but as a side note: If the clipboard data source app needs to delay render the clipboard data, it's less likely that a slow strlen() in the app reading the clipboard will account for a significant part of the time the clipboard lock is held.

Reply:
Yeah, I also benchmarked it, and for any realistic clipboard size (<100kB) it really doesn't matter at all. I'll remove it to keep the code base lean. But it does make a difference, even if it is small, due to the overall overhead of getting the clipboard content. It's about ~10x faster than the intrinsic `wcslen`.